IbisPlugins/GPU_RigidRegistration/gpu_rigidregistrationwidget.cpp

```diff
@@ -151,12 +153,13 @@ void GPU_RigidRegistrationWidget::UpdateUi()
   ui->initialSigmaComboBox->addItem( "8.0", QVariant( 8.0 ) );
   ui->initialSigmaComboBox->setCurrentIndex( 1 );
 
-  SceneManager * sm = m_pluginInterface->GetSceneManager();
-  const SceneManager::ObjectList & allObjects = sm->GetAllObjects();
+  IbisAPI *ibisAPI = m_pluginInterface->GetIbisAPI();
+  Q_ASSERT(ibisAPI);
+  const QList< SceneObject* > allObjects = ibisAPI->GetAllObjects();
```

Comment:
GetAllObjects returns a reference to a list, not the list itself. It will work anyways, but will create a copy of the list instead of using the original as in the old code.